site/src/templates/standard-article.js

```diff
@@ -53,8 +55,12 @@ export default class Article extends React.Component {
       anchorScroll(history)
     }
 
+    let paginationObject = new PaginationObject()
```

Comment:
Would prefer paginator to paginationObject.  ...Object is redundant

Reply:
Could you please add a test for this.

Reply:
Done